### ISSUE TITLE
React Package Update

### ISSUE THREAD

**Issue by zdizzle6717** [benwiley4000/react-responsive-audio-player#29]
This has been a really useful module, thank you for that.  With recent updates to react, PropTypes will soon be deprecated in favor of a separate npm package, prop-types.

Could you update the current build to use prop-types?

Here's a quick explanation of what's new in React...soon to be React 16.0.0
https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes

**Comment by benwiley4000** [comment]
@zdizzle6717 thanks for flagging this! While I could try to find time to get to this soon, would you like to submit a pull request for it instead?

**Comment by zdizzle6717** [comment]
I not quite sure how a pull request works.  Could you briefly explain?

**Comment by zdizzle6717** [comment]
How can I push a topic branch to compare with master in the new pull request

**Comment by benwiley4000** [comment]
Yep! To make a pull request, you can fork this repository, make your changes on a feature branch, then hit "new pull request" on this repository's page - you should set the "base" to "master" and the "compare" branch would be your new feature branch on your forked repository. There should be a button to automatically do that configuration for you if you've pushed your changes recently.

Make sure to include "For #29" in the PR description.

I can think of a few needed steps for this one:
1. Add `prop-types` to the package.json `dependencies`, which you'll need to create (as opposed to `peerDependencies` - as pointed out [here](https://github.com/reactjs/prop-types/issues/44), there's no reason to always expect the user to specify their own version of `prop-types`).
2. Make `prop-types` an external, like React, in the webpack configuration (we want the user's `node_modules` to provide this package during their compile step, not ours).
3. Change references in `src/index.js`.

We may want to add version 16 to our allowed React versions, but we could wait on that until it's released/verified to work.

We'll also want to make sure the example runs without errors - see development instructions in [CONTIBUTING.md](CONTRIBUTING.md).

**Comment by benwiley4000** [comment]
Hey @zdizzle6717 - do you plan to work on a PR?

**Comment by zdizzle6717** [comment]
Thanks for the quick tutorial.  Let me know what happens next.
